misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/utils/BalFileGenerationUtils.java

```diff
@@ -90,6 +92,29 @@ public static void generateDescriptor(String command) {
             throw new BalGenToolException("Process not successfully completed. Process is interrupted while" +
                     " running the protoC executor.", e);
         }
+        if (process.exitValue() != 0) {
+            BufferedReader br = null;
+            try {
+                br = new BufferedReader(new
+                        InputStreamReader(process.getErrorStream(), "UTF-8"));
+                String err;
+                StringBuilder errMsg = new StringBuilder(EMPTY_STRING);
+                while ((err = br.readLine()) != null) {
+                   errMsg.append(System.lineSeparator().concat(err));
```

Comment:
Shall we change this to,
```java
                StringBuilder errMsg = new StringBuilder();
                while ((err = br.readLine()) != null) {
                   errMsg.append(System.lineSeparator()).append(err);
                }
```

Reply:
Fixed in 75bd97a685bc547f9e11e8ca2c4107ed470c3bfb